cmd/minishift/cmd/config/unset.go

```diff
@@ -44,10 +44,18 @@ func init() {
 }
 
 func unset(name string) error {
+	_, err := findSetting(name)
+	if err != nil {
+		return err
+	}
 	m, err := ReadConfig()
 	if err != nil {
 		return err
 	}
+	if m[name] == nil {
+		return fmt.Errorf("Property name '%s' is not set", name)
+	}
 	delete(m, name)
+	fmt.Printf("Property name '%s' successfuly unset\n", name)
```

Comment:
@hferentschik I remember you are saying providing output to all commands related for saying success was counter productive to you. I am not sure why we came to that conclusion. Can you take a look here?

Reply:
@LalatenduMohanty my understanding is we should only provide direct feedback related to unset command when a user try to unset a config which either not set or exist. For successfully unset we don't need to include this message.